download-basket/src/main/java/downloadbasket/data/ZipDownloadDetails.java

```diff
@@ -0,0 +1,21 @@
+package downloadbasket.data;
+
+/**
+ * Created by markokuo on 6.10.2015.
+ */
+public class ZipDownloadDetails {
+    private String FileName = "";
```

Comment:
Use lowercase first letter for variables

Reply:
fixed